### PR TITLE
Add touble shooting

### PR DESCRIPTION
**Pull request by spearkkk**
Hi, this is spearkk.
I appreciate your nice sharing.

Actually I do not have synlink file for `/usr/local/bin/node` because I use `nvm`.
So there is a issue to use this nice tool.
Even if there is nice notice message in your workflow, I cannot solve this issue.

For others like me, I just add touble shooting menu.
I tried to doc for contribution, but I couldn't find it.

So I just fork -> edit -> send PR to you.

If you have any concern or request, idea, let me know.
Thank you.

### CODE REVIEW

**Comment by bchatard** [comment]
Hi,

thanks for your contribution
Can you please move the troubleshooting section into a dedicated file in `doc` folder?
Maybe something like this `doc/troubleshooting.md`
And put your guide under an "NVM" section :)

**Comment by spearkkk** [comment]
@bchatard  
I moved it to `doc/trouble_shooting.md`.
Please review again this PR.
Thank you

File: doc/trouble_shooting.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Trouble Shooting
```
Comment (bchatard) [inline doc/trouble_shooting.md:1]:
```suggestion
# Troubleshooting
```

File: doc/trouble_shooting.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+---
+## NVM
+
+- If you get this message: 'Couldn't find the `node` binaray',  
+ You should create symbolic link file in `/usr/local/bin`.
+    
+    Actually I manage node using `nvm`, so I just did:  
+    ```shell  
+      sudo ln -s `which node` /usr/local/bin/node  
+    ```
```
Comment (bchatard) [inline doc/trouble_shooting.md:13]:
```suggestion
If you get this message: "Couldn't find the `node` binary".
It's maybe du to non existing node version available globally.
You should create symbolic link file in your PATH (eg: `/usr/local/bin`).
```shell  
$ sudo ln -s $(which node) /usr/local/bin/node
\```
```

**Comment by spearkkk** [comment]
Thanks for your help :D

**Pull request merged**